src/home/spaces_dock.rs

```diff
@@ -137,16 +180,77 @@ live_design! {
             <Filler> {}
 
             <Profile> {}
-    
+
             <Filler> {}
-    
+
             <Home> {}
-            
+
             <Filler> {}
-    
+
             <Settings> {}
-    
+
             <Filler> {}
         }
     }
 }
+
+#[derive(Live, LiveHook, Widget)]
+pub struct Profile {
+    #[deref]
+    view: View,
+    #[rust(VerificationState::Unknown)]
+    verification_state: VerificationState,
+}
+
+impl Widget for Profile {
+    fn handle_event(&mut self, cx: &mut Cx, event: &Event, scope: &mut Scope) {
+        if let Event::MouseMove(e) = event {
+            if self.view(id!(verification_icon)).area().rect(cx).contains(e.abs) {
+                let text = match self.verification_state {
+                    VerificationState::Unknown => {"Unknown"},
+                    VerificationState::Unverified => {"Unverified"},
+                    VerificationState::Verified => {"Verified"}
```

Comment:
This is a good start, but it doesn't really provide much more information to the user than the bare icon does. Let's add some more detail.

```suggestion
                    VerificationState::Unknown => {"Verification state is unknown."},
                    VerificationState::Unverified => {"This device is unverified. To view your encrypted message history, please verify it from another client."},
                    VerificationState::Verified => {"This device is fully verified."}
```